### PR TITLE
feat(version): Bump DF version to `v1.11.0`

### PR DESCRIPTION
**Pull request by Pothulapati**

### CODE REVIEW

**Comment by ashotland** [comment]
Only released on Monday - don't you want to give it a few more days?

**Comment by Pothulapati** [comment]
@ashotland Sounds like a good idea! Raised as #117 depends on this but we can work on it independently. Will aim for next week for both of these PR's.

File: e2e/dragonfly_controller_test.go [hunk 1]
```diff
@@ -295,7 +295,7 @@ var _ = Describe("Dragonfly Lifecycle tests", Ordered, FlakeAttempts(3), func()
 		})
 
 		It("Update to image should be propagated successfully", func() {
-			newImage := resources.DragonflyImage + ":v1.9.0"
+			newImage := resources.DragonflyImage + ":v1.10.0"
```
Comment (Abhra303) [inline e2e/dragonfly_controller_test.go:298]: Shouldn't the version be "v1.11.0"?

**Comment by Pothulapati** [comment]
Closing in favour of https://github.com/dragonflydb/dragonfly-operator/pull/128

**Pull request closed without merging**